AzureIndexer/Stratis.Bitcoin.Features.AzureIndexer/AzureIndexerLoop.cs

```diff
@@ -244,7 +244,8 @@ private BlockFetcher GetBlockFetcher(IndexerCheckpoints indexerCheckpoints, Canc
                         task.SaveProgression = !this.indexerSettings.IgnoreCheckpoints;
                         var fetcher = this.GetBlockFetcher(IndexerCheckpoints.Blocks, cancellationToken);
                         task.Index(fetcher, this.AzureIndexer.TaskScheduler);
-                        minHeight = Math.Min(minHeight, fetcher._LastProcessed.Height);
+                        if (fetcher._LastProcessed != null)
+                            minHeight = Math.Min(minHeight ?? int.MaxValue, fetcher._LastProcessed.Height);
```

Comment:
looks like minHeight is always going to be null here

Reply:
Math.Min(minHeight <- here

Reply:
You are right. I have refactored the code.